homeassistant/components/meteo_france/sensor.py

```diff
@@ -48,12 +75,25 @@ def state(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes of the sensor."""
+        # Attributes for next_rain sensor.
         if self._condition == 'next_rain' and 'rain_forecast' in self._data:
             return {
                 **{STATE_ATTR_FORECAST: self._data['rain_forecast']},
                 ** self._data['next_rain_intervals'],
                 **{ATTR_ATTRIBUTION: ATTRIBUTION}
             }
+
+        # Attributes for weather_alert sensor.
+        if self._condition == 'weather_alert' \
+                and self._alert_watcher is not None:
+            return {
+                **{STATE_ATTR_BULLETIN_TIME:
+                   self._alert_watcher.bulletin_date},
+                ** self._alert_watcher.alerts_list,
```

Comment:
If this is a list, there should only be one star to expand it, I think. And if it's a list it can't be unpacked into a dict. Am I missing something?

Reply:
You are right alert_list attribute is a dict.